### PR TITLE
TCVP-2702 passed isEmailVerified through to Update Dispute form

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

TCVP-2702
Change of requirements - On the Update Dispute Request screen, the opting out should be based on whether the user had previously opted out of email or not.
- modified DisputeResult.isEmailAddressVerified to be a tri-state variable (null for opted out, true for verified, false for not verfied)
- updated openapi client in .NET
- passed isEmailAddressVerified through to Update Dispute screen and defaulted the opted out based on that.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change
- [x] Change request

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by saikrishnametpalli-nttdata — APPROVED** [review]

File: src/frontend/citizen-portal/src/app/components/update-dispute-contact/update-dispute-contact.component.ts [hunk 1]
```diff
@@ -17,14 +18,18 @@ export class UpdateDisputeContactComponent implements OnInit {
...
 
   constructor(
     private disputeService: DisputeService,
     private store: Store,
+    private route: ActivatedRoute
   ) {
   }
 
   ngOnInit(): void {
+    this.preferEmail = this.route.snapshot.queryParamMap.get('preferEmail') === null ? false : this.route.snapshot.queryParamMap.get('preferEmail') === 'true';
```
Comment (Sharkww) [inline src/frontend/citizen-portal/src/app/components/update-dispute-contact/update-dispute-contact.component.ts:31]: `this.preferEmail = this.route.snapshot.queryParamMap.get('preferEmail') === 'true' ? true : false;`

File: src/frontend/citizen-portal/src/app/components/update-dispute-landing/update-dispute-landing.component.ts [hunk 2]
```diff
@@ -61,6 +61,6 @@ export class UpdateDisputeLandingComponent implements OnInit {
   }
 
   goToUpdateDisputeContact(): void {
-    this.disputeService.goToUpdateDisputeContact(this.state.params);
+    this.disputeService.goToUpdateDisputeContact(this.state.params, this.isEmailVerified === null ? false : true);
```
Comment (Sharkww) [inline src/frontend/citizen-portal/src/app/components/update-dispute-landing/update-dispute-landing.component.ts:64]: `this.disputeService.goToUpdateDisputeContact(this.state.params, this.isEmailVerified !== null);`

**Review by karahanb — APPROVED** [review]
Great work!

File: src/frontend/citizen-portal/src/app/components/update-dispute-landing/update-dispute-landing.component.ts (hunk 2 above)
Comment (KDNeufeld) [reply src/frontend/citizen-portal/src/app/components/update-dispute-landing/update-dispute-landing.component.ts]: done

File: src/frontend/citizen-portal/src/app/components/update-dispute-contact/update-dispute-contact.component.ts (hunk 1 above)
Comment (KDNeufeld) [reply src/frontend/citizen-portal/src/app/components/update-dispute-contact/update-dispute-contact.component.ts]: done.

**Review by Sharkww — APPROVED** [review]

**Pull request merged**